### PR TITLE
Add missing directory separator

### PR DESCRIPTION
**Pull request by dharmatech**
Before:

image

After:

image

### CODE REVIEW

File: aspnetcore/data/ef-rp/migrations.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Run the app and verify that the database is seeded.
 
 ## Up and Down methods
 
-The EF Core `migrations add` command generated code to create the database. This migrations code is in the *Migrations\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the database tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
+The EF Core `migrations add` command generated code to create the database. This migrations code is in the *Migrations/\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the database tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
```
Comment (Rick-Anderson) [inline aspnetcore/data/ef-rp/migrations.md:97]:
```suggestion
The EF Core `migrations add` command generated code to create the database. This migrations code is in the `Migrations\<timestamp>_InitialCreate.cs` file. The `Up` method of the `InitialCreate` class creates the database tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
```

File: aspnetcore/data/ef-rp/migrations.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ dotnet ef database update
 
 ### Examine the Up and Down methods
 
-The EF Core `migrations add` command  generated code to create the DB. This migrations code is in the *Migrations\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the DB tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
+The EF Core `migrations add` command  generated code to create the DB. This migrations code is in the *Migrations/\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the DB tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
```
Comment (Rick-Anderson) [inline aspnetcore/data/ef-rp/migrations.md:226]:
```suggestion
The EF Core `migrations add` command generated code to create the database. This migrations code is in the *Migrations\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the database tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@dharmatech  thanks! Can you check my commit. Your approach works but code fencing is simpler.

File: aspnetcore/data/ef-rp/migrations.md [hunk 3]
```diff
@@ -223,7 +223,7 @@ dotnet ef database update
 
 ### Examine the Up and Down methods
 
-The EF Core `migrations add` command  generated code to create the DB. This migrations code is in the *Migrations\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the DB tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
+The EF Core `migrations add` command generated code to create the database. This migrations code is in the *Migrations\<timestamp>_InitialCreate.cs* file. The `Up` method of the `InitialCreate` class creates the database tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
```
Comment (Rick-Anderson) [inline aspnetcore/data/ef-rp/migrations.md:226]:
```suggestion
The EF Core `migrations add` command generated code to create the database. This migrations code is in the `Migrations\<timestamp>_InitialCreate.cs` file. The `Up` method of the `InitialCreate` class creates the database tables that correspond to the data model entity sets. The `Down` method deletes them, as shown in the following example:
```

Comment (dharmatech) [reply aspnetcore/data/ef-rp/migrations.md]: Looks good. Thanks Rick!

**Pull request merged**